src/use-swr.ts

```diff
@@ -324,7 +329,17 @@ export const useSWRHandler = <Data = any, Error = any>(
       // Here is the source of the request, need to tell all other hooks to
       // update their states.
       if (isCurrentKeyMounted() && shouldStartNewRequest) {
-        broadcastState(cache, key, newState.data, newState.error, false)
+        updatedAtRef.current = Date.now()
+        broadcastState(
+          cache,
+          key,
+          newState.data,
+          newState.error,
+          false,
+          false,
+          undefined,
+          getUpdatedAt()
```

Comment:
Yeah, this turned out to look rather ugly, but I didn't want to change the existing order of parameters, though, updatedAt, arguably has more importance than the rest, as it is consumed by users.